doc/manual/generate-builtins.nix

```
@@ -1,24 +1,30 @@
 let
-  inherit (builtins) concatStringsSep attrNames;
+  inherit (builtins) concatStringsSep attrValues mapAttrs;
+  inherit (import ./utils.nix) optionalString squash;
 in
 
 builtinsInfo:
 let
-  showBuiltin = name:
+  showBuiltin = name: { doc, args, arity, experimental-feature }:
     let
-      inherit (builtinsInfo.${name}) doc args;
+      xpNotice = optionalString (experimental-feature != null) ''
+        This function is only available if you enable the
+        [experimental](@docroot@/contributing/experimental-features.md) feature
+        [${experimental-feature}](@docroot@/contributing/experimental-features.md#xp-feature-${experimental-feature}.
```

We should assume that we will implement https://github.com/NixOS/nix/issues/8466 soon-ish, and that link will break nicely to prompt for updating it appropriately.

```suggestion
        This function is only available if the [${experimental-feature}](@docroot@/contributing/experimental-features.md#xp-feature-${experimental-feature}) experimental feature is enabled.
```